UnityProject/Assets/Scripts/UI/Systems/Jobs/GUI_PlayerJobs.cs

```diff
@@ -149,6 +149,7 @@ private void OnDisable()
 		{
 			UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
 			expirementalJobsTestToggle.isOn = false;
+			footer.DestroyAllChildren();
```

Comment:
why DestroyAllChildren?

Reply:
because it creates duplicates when you enable the footer again